setup.py

```diff
@@ -20,7 +20,7 @@ def readme():
                   "interactive, publication-quality graphs.",
       long_description=readme(),
       classifiers=[
-          'Development Status :: 4 - Beta',
```

Comment:
I don't know that it makes sense to upgrade this from beta to stable on a minor version release. Maybe we should wait for 2.x. I think it would be a good idea to have a meeting on goals for 2.x with this switch from Beta to Production in mind?

I also don't know how much weight this little line holds, so I'm not that worried about it.